cli/tests/integrations/common.py

```diff
@@ -30,7 +30,7 @@ def exec_command(cmd, env=None, stdin=None):
         stderr=subprocess.PIPE,
         env=env)
 
-    stdout, stderr = process.communicate()
+    stdout, stderr = [_.replace(b'\r', b'') for _ in process.communicate()]
```

Comment:
underscore is for variables you don't use.  you use this one.  name it something else